felles/src/test/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/service/ServiceRiverStatefulTest.kt

```diff
@@ -352,89 +350,6 @@ class ServiceRiverTest :
             }
         }
 
-        test("datamelding trigger ikke service ved manglende startdata (inaktiv service)") {
```

Comment:
`ServiceMedXSteg` har overtatt ansvaret for å sjekke for inaktivitet, så denne og følgende tester er ikke lenger aktuelle.